examples/hybrid/driver.jl

```diff
@@ -364,7 +364,6 @@ if parsed_args["debugging_tc"]
     )
 
     main_branch_root = get_main_branch_buildkite_path()
-    @info "Comparing PR profiles against main with commit id: $(basename(main_branch_root))"
```

Comment:
Was this deletion intentional? I didn't write this, but these can be useful breadcrumbs when looking back through logs

Reply:
This isn't a deletion; it's a [move](https://github.com/CliMA/ClimaAtmos.jl/blob/fe3b9d47c74b15304723c00a32561c534ca66f0d/regression_tests/self_reference_or_path.jl#L120-L121).